sparse/_compressed/compressed.py

```diff
@@ -115,6 +116,9 @@ def __init__(
         self, arg, shape=None, compressed_axes=None, prune=False, fill_value=0
     ):
 
+        if isinstance(arg, ss.spmatrix):
+            arg = self.from_scipy_sparse(arg)
+
```

Comment:
Can we make `scipy` a soft requirement here?

Reply:
Two things:

* Do you have a preferred way to make dummy imports in this package?
* I think it'd be really nice to be able to lean on `scipy` to get these classes feature complete. Is there a rush to remove it as a dependency before then?

Reply:
> Two things:
> 
>     * Do you have a preferred way to make dummy imports in this package?
> 
>     * I think it'd be really nice to be able to lean on `scipy` to get these classes feature complete. Is there a rush to remove it as a dependency before then?

Not really, but I was avoiding making it harder when we do do it, there are very few hard links to it now. But I agree, the path of least resistance can be taken here.